### ISSUE TITLE
Version revision bump?

### ISSUE THREAD

**Issue by 01micko** [labwc/labwc-tweaks#27]
First, thanks @johanmalm for this tool, and allowing me to cooperate in it's development.

I think it might be time to bump the version to `0.1.1` unless you or anyone else has some fairly immediate and worthy improvements or bugfixes in the pipeline. 

Thoughts?

**Comment by 01micko** [comment]
One thing I wouldn't mind adding, but not too soon, is Native Language Support. It can wait until I have more time.

**Comment by johanmalm** [comment]
Well, I do have one dilemma.

A tag signals to maintainers that it's reached some sort of maturity and that they could consider packaging it. And all of a sudden I feel a responsibility to maintain backward compatibility, etc.

I feel that before we do anything else (like tagging or add more features), we ought to re-write it with a ui-file because it scales better.

...which makes me think we really ought to move to gtk4 or Qt6 first to avoid re-work. I've had a go at gtk4 already, but it's pretty hard because even to just do a dropdown menu that's slightly more complex than a string-list you have to create a data object and expose some properties (which I've just about got my head around). But then you have to put them in a `GListStore`... and then create a List Item Factory and create callbacks for setup/bind signals... and I have to admit I found it a bit hard and not managed to get it to work. There are lots of python and rust examples, but not many for C.

I might persevere with it, but may just switch to Qt because I find that easier. A port to Qt would probably take me the best part of a day (which is time I rarely have) but have scheduled in for March (https://github.com/johanmalm/roadmap/issues/1).

So, in summary I prefer to port to gkt4/Qt6, then stabilize, then tag. Which looks like May(ish).

**Comment by 01micko** [comment]
Thanks for the response.

Yeah, I understand time constraints only too well. I'll close this as it's not a priority.

**Comment by SpidFightFR** [comment]
Subbing to this issue, if everything changes in the future, please tell !
